components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/applications/payload/BasicPayloadData.java

```diff
@@ -85,8 +85,8 @@ public void populatePayload () {
         payloadBuilder.append("CARTRIDGE_KEY=" + getSubscriptionKey());
         payloadBuilder.append(",");
         payloadBuilder.append("DEPLOYMENT=" + AutoScalerConstants.PAYLOAD_DEPLOYMENT);
-        payloadBuilder.append(",");
-        payloadBuilder.append("APPLICATION_PATH=" + getApplicationPath());
+//        payloadBuilder.append(",");
```

Comment:
I think this is not correct. We will need to add APPLICATION_PATH to the basic payload if base dir has been defined in the cartridge. If not Kubenetes workflow will break.
Thanks